packages/theming/theme-types/src/Theme.types.ts

```diff
@@ -25,7 +25,7 @@ export interface Theme {
     [key: string]: object; // eslint-disable-line @typescript-eslint/ban-types
   };
   shadows: ThemeShadowDefinition;
-  spacing: Spacing;
+  readonly spacing: Spacing;
```

Comment:
Apologies for putting this up in a merge code, but I want to understand if this can break an existing partner overriding Spacing till now...? Do we have any that we are aware.?

Reply:
Post-merge comments are fine @ankraj12

While yes, this change is technically breaking, I don't think anyone really uses the spacing values from the Theme or overrides them so I think this is fine. We can always limit the readonly to just the new entries if we get feedback.